livekit-agents/livekit/agents/pipeline/pipeline_agent.py

```diff
@@ -853,6 +853,9 @@ async def _llm_stream_to_str_generator(
         ) -> AsyncGenerator[str]:
             try:
                 async for chunk in stream:
+                    if not chunk.choices:
```

Comment:
was this happening?

Reply:
It can if we receive a chunk that is only for CompletionUsage